src/main/java/io/zeebe/monitor/zeebe/KafkaStreamsService.java

```diff
@@ -0,0 +1,80 @@
+package io.zeebe.monitor.zeebe;
+
+import io.zeebe.monitor.zeebe.kafka.ZeebeKafkaStreams;
+import io.zeebe.monitor.zeebe.kafka.importers.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+@Component
+public class KafkaStreamsService {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+    @Autowired
+    private KakaProcessAndElementImporter processAndElementImporter;
+
+    @Autowired
+    private KafkaTimerImporter timerImporter;
+
+    @Autowired
+    private KafkaVariableImporter variableImporter;
+
+    @Autowired
+    private KafkaErrorImporter kafkaErrorImporter;
+
+    @Autowired
+    private KafkaMessageImporter kafkaMessageImporter;
+
+    @Autowired
+    private KafkaIncidentImporter kafkaIncidentImporter;
+
+    @Autowired
+    private ZeebeKafkaStreams zeebeKafkaStreams;
+
+    @PostConstruct
+    public void start() {
+        zeebeKafkaStreams.kafkaProcessStream().foreach((key, event) -> {
+            if (event != null) {
```

Comment:
can event actually be null?

Reply:
Event may not be null but during conversion from json to java object exception may happen